### PR TITLE
add 2021.bib and add one

### PR DESCRIPTION
**Pull request by tjhei**
I will update the .include separately.

### CODE REVIEW

**Comment by tjhei** [comment]
Actually, I will just include it here as well.

File: publications.include [hunk 1]
```diff
@@ -69,7 +69,7 @@
               <strong>The <abbr>deal.II</abbr> finite element library: design, features, and insights
               </strong>
               <br>
-              Computers &amp; Mathematics with Applications, 2020.
+              Computers &amp; Mathematics with Applications, 2021.
```
Comment (bangerth) [inline publications.include:72]: Can you add volume and page numbers?

**Review by bangerth — APPROVED** [review]
Please add the one missing piece of information, and then merge yourself.

Separately: Did we ever announce the existence of this publication on the mailing list?

**Comment by tjhei** [comment]
Done.

> Separately: Did we ever announce the existence of this publication on the mailing list?

I just did.

**Review by bangerth — APPROVED** [review]

**Pull request merged**